src/pages/Registration/Signup/Form.tsx

```diff
@@ -23,7 +23,7 @@ export default function Form({ classes = "" }: { classes?: string }) {
   return (
     <form
       onSubmit={submit}
-      className={`${classes} padded-container w-full max-w-[37.5rem] grid`}
+      className={`${classes} justify-center gap-8 padded-container w-full max-w-[37.5rem] grid`}
     >
       <h3 className="text-3xl font-bold text-center">
         {`Register to ${APP_NAME}`}
```

Comment:
The "Connect wallet" section is missing.
![image](https://user-images.githubusercontent.com/19427053/226832146-f3bb96c4-f922-45a2-baa2-84c6b12253c7.png)

Do we not need this @SovereignAndrey ? Users occasionally get confused and already registered users might find this tooltip/section useful in remembering what they need to do to "sign in".

Reply:
Good catch! Yes we need that in there.

Reply:
@misicnenad what if user has already connected a wallet, do we display this statement then too?

Reply:
These are all great points @abbasali1994 & @misicnenad. I think, for now, we can revert this last commit, leaving the connect wallet out, as it might cause more confusion with it in there than not. It doesn't actually do **anything** at this stage (though maybe we might want to pull applications by that wallet in the future that are pending and show them a list?). I'll discuss with Lambert tomorrow and circle back. :)